### PR TITLE
[falco-security] Check availability of 'monitoring.coreos.com/v1' api version

### PR DESCRIPTION
**Pull request by yasintahaerol**
**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

> /kind bug

> /kind cleanup

> /kind design

> /kind documentation

> /kind failing-test

/kind feature

> If this PR will release a new chart version please make sure to also uncomment the following line:

> /kind chart-release

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

> /area falco-chart

> /area falco-exporter-chart

> /area falcosidekick-chart

**What this PR does / why we need it**:

To avoid getting an error when deploying the falco-exporter with a helm chart when there is no prometheus operator in your k8s cluster.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Checklist**

- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] CHANGELOG.md updated

### CODE REVIEW

**Comment by yasintahaerol** [comment]
Hey. Thanks for the feedback. I signed off all commits.

File: falco-exporter/CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,12 @@ numbering uses [semantic versioning](http://semver.org).
 
 ## v0.6.1
```
Comment (developer-guy) [inline falco-exporter/CHANGELOG.md:6]: it should be 0.6.2, right?

**Review by developer-guy — APPROVED** [review]
LGTM

**Comment by developer-guy** [comment]
PTAL @Issif 🙋🏻‍♂️🙏

**Review by cpanato — APPROVED** [review]
/lgtm

**Pull request merged**